packages/web/src/components/data-entry/InputV2.tsx

```diff
@@ -27,6 +28,7 @@ export type InputV2Props = Omit<ComponentPropsWithoutRef<'input'>, 'size'> & {
   error?: boolean
   inputClassName?: string
   label?: string
+  helperText?: string
 }
 
 export const InputV2 = (props: InputV2Props) => {
```

Comment:
Should we name this HarmonyInput? Or is it not up to that standard?

Reply:
yeah good q... i would personally like to do another pass with everything before moving it over (adding forwardRef etc)